### DISCUSSION TITLE
Cant find files on old drive

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/WSL#6764] [Q&A]
I got a new ssd for my pc and I installed a fresh copy of windows. I went to go boot to my old ssd and it errors out.(I have tried everything) and I have projects on there i need to access so i went to the appdata\local\microsoft\windowsapps\canonicalgroup... and there is only a ubuntu2004.exe and nothing else. Am I still able to get my files back?

**Comment by therealkenc** [comment]
You might (if you are fortunate) find the vhdx in:
```
C:\Users\you\AppData\Local\Packages\CanonicalGroupLimited....\LocalState
```

**Reply by ghost** [reply]
it worked thank you
